routers/web/explore/user.go

```diff
@@ -62,16 +62,31 @@ func RenderUserSearch(ctx *context.Context, opts *models.SearchUserOptions, tplN
 		orderBy = models.SearchOrderByAlphabetically
 	}
 
+	statusFilterKeys := []string{"is_active", "is_admin", "is_restricted", "is_2fa_enabled", "is_prohibit_login"}
```

Comment:
I will say no. You have to check the permissions of login user. Whether he can use the filter. Is a plain user could know who are admin? Who are active? etc.

Reply:
Fixed. Now only in admin/users.go can use these filters.